src/Compiler/DependencyManager/DependencyProvider.fs

```diff
@@ -160,15 +160,15 @@ type ReflectionDependencyManagerProvider
 
     let instance =
         if not (isNull (theType.GetConstructor([| typeof<string option>; typeof<bool> |]))) then
-            Activator.CreateInstance(theType, [| outputDir :> objnull; useResultsCache :> objnull |])
+            Activator.CreateInstance(theType, [| outputDir :> obj | null; useResultsCache :> obj | null |])
```

Comment:
`objnull` is an alias which became part of the public API of FSharp.Core -> that one can remain.
It is just an alias to obj|null, both are fine.